erpnext/selling/doctype/selling_settings/selling_settings.json

```diff
@@ -25,7 +25,7 @@
   "editable_price_list_rate",
   "validate_selling_price",
   "editable_bundle_item_rates",
-  "transaction_settings_section",
+  "sales_transactions_settings_section",
```

Comment:
The section break fieldname is not changed.

Reply:
Also, the modified timestamp should also change, else these changes won't reflect in any site instance

Reply:
Done.